src/com/hms_networks/americas/sc/thingworx/config/TWConnectorConfig.java

```diff
@@ -173,6 +173,56 @@ public String getThingworxAppKey() throws JSONException {
     return twAppKey;
   }
 
+  /**
+   * Get the number of maximum data points in a payload from the configuration.
+   *
+   * @return maximum data points in a payload
+   * @throws JSONException if unable to get maximum data points in a payload from the configuration
```

Comment:
Change throws description, its slightly misleading. If max data points is missing from config file (unable to get), constant will be used. Change description to: "unexpected JSON parsing error"

OR put try/catch in this function.

Reply:
updated description